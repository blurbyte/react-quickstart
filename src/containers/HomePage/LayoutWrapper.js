import styled from 'styled-components';
import media from '../../styles/media';

const LayoutWrapper = styled.div`
  display: flex;
  justify-content: space-between;
  ${media.tablet`display: block;`}
`;

export default LayoutWrapper;
