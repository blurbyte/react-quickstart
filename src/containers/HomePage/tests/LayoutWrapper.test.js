import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import LayoutWrapper from '../LayoutWrapper';

describe('<AppWrapper />', () => {
  it('should render correctly', () => {
    // component init
    const component = renderer.create(<LayoutWrapper />);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
