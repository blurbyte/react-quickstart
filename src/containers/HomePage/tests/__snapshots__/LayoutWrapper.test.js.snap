// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppWrapper /> should render correctly 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: space-between;
  -webkit-justify-content: space-between;
  -ms-flex-pack: space-between;
  justify-content: space-between;
}

@media (max-width:700px) {
  .c0 {
    display: block;
  }
}

<div
  className="c0"
/>
`;
